mono/utils/mono-threads.h

```diff
@@ -714,6 +719,8 @@ int mono_thread_info_suspend_count (THREAD_INFO_TYPE *info);
 int mono_thread_info_current_state (THREAD_INFO_TYPE *info);
 const char* mono_thread_state_name (int state);
 gboolean mono_thread_is_gc_unsafe_mode (void);
+G_EXTERN_C // due to THREAD_INFO_TYPE varying
```

Comment:
Ludovic wants `G_EXTERN_C` on same line going forward.
I guess I should have eliminated `THREAD_INFO_TYPE` by now.

Reply:
@jaykrell I never said that as a general rule, I certainly said to keep consistent with the surrounding context though  :wink: